design-system/materials/internals/definition.yaml

```diff
@@ -597,6 +607,8 @@ decisionGroupsByTheme:
           choice: shadow-0
         shadow-for-button-when-disabled:
           choice: shadow-0
+        shadow-for-card-when-raised:
+          choice: 'none'
```

Comment:
This token is in the new theme group so we should use the new value provided by UX: `0 1px 2px rgb(25 40 81 / 5%)`.